guardian/newsletters-nx
# Add derived values steps to launch wizard

## What does this change?

The steps for viewing and (optionally) editing the derived values (identity name, braze variables, ophan variables) were removed from the initial newsletterData wizard: this PR adds those pages into the Launch wizard.

## How to test

Run `npm run dev`
From the list of drafts, select a draft and enter all mandatory fields (5770 Avon Recumbent is already fully populated)
Click the Launch button and navigate through the wizard
Ensure that it works both if selecting Next all the way through, and also if selecting any of the Edit options
Check that editing data on other wizards still works (as a shared function has been changed)

## How can we measure success?

Data changes, if made, are saved
After pressing Launch at the end of the wizard, the newsletter is moved from the list of Drafts to the list of Launched

## Have we considered potential risks?

This wizard does not allow the user to skip steps
Furthermore, a Back button is available only on the pages showing derived fields (otherwise, it is not possible to navigate backwards and forwards)
The user needs to complete the Launch wizard steps in order

The derivation of the fields from the newsletter name needs to be moved from its current position, but this will be addressed in a separate PR

## Images

(Note that this was recorded before merging from main)

https://user-images.githubusercontent.com/74301289/235904366-896601d2-6ed0-44d2-a170-85112d85a293.mov

## Accessibility
